docs/source/checks/simple.rst

```diff
@@ -0,0 +1,175 @@
+
+    (...)
+
+
+
+Interpretation
+++++++++++++++
+
+The results show column-wise wich shifts appear in your dataset. First the reports of your numerical columns are shown,
+after that the reports of your categorical columns.
+
+Numerical
+~~~~~~~~~
+
+In the numerical columns you see that the column with the name 'payment' is detected as shifted. You see below
+the columnname that two numerical metrics detected a shift, *mean* and *std* (standard deviation).
+
+*Diff* describes the relative value-distance of the metrics that are calculated on your two datasets. If the Diff
+exceeds a metric-specific threshold, the dataset differ that much, that the check calls shift on this column.
+
+In this case, the mean-value of 'payment' could be 1000.0 in your first, but 706.0 in your second dataset. This difference is
+results in an difference of -294 and an relative difference of -29.4%. The threshold set here is 20% which is exceeded
+by the absolute value of -29.4%. All thresholds are customizable through the API of the
+:class:`~shift_detector.checks.simple_check.SimpleCheck`. All the metrics are listed in the section
+:ref:`simple_check_parameters`.
+
+Categorical
+~~~~~~~~~~~
+
+Using the SimpleCheck shifts can also be detected on categorical columns. Here the histograms over the attribute-values are
+compared. If the difference between those values exceeds the *categorical_threshold* the check calls shift on this
+column.
+
+The example above shows a shift in the categorical column 'payment_option'. There are different
+attribute-values in this column, i.e. 'debit and 'cash'. In those, the differences between the dataset are 6% and
+-6.5% which both exceed the threshold of 5%. This indicates that the in dataset2 more people use
+cash as a payment-option and fewer use debit, the check calls shift on the column.
+
+
+.. _simple_check_parameters:
+
+Metrics & Parameters
+----------
+
+There are 9 different numerical metrics in the SimpleCheck whose differences can indicate a shift. All
+default-thresholds can be adjusted.
+
++-----------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------+
+| metric_name                       |                                                                           explanation                                                   |
++===================================+=========================================================================================================================================+
+| **quartile_1,                     |                                                                                                                                         |
+| median,                           | Those are the .25- .5- and .75-quantiles of the column.                                                                                 |
```

Comment:
is quartile an quantile the same? Can we use consistent names?

Reply:
Here I use the more common term 'quantile' to describe the term 'quartile'. A the quartiles are the quantiles that are dividable by 0.25